### DISCUSSION TITLE
Icon Pack for Nextjs

### DISCUSSION THREAD

**Discussion by IBNAHMADMOHAMED1** [vercel/community#1120] [Show & Tell]
Hi  ,

I am a stu­dent of web dev­elop­ment and I have created a pack­age of icons for the nextjs. These icons are for use in web pages, ap­pli­ca­tions and tools. [link](https://icon-react.vercel.app/)

I would be very happy if you could give your feedback about this package.

This package includ­es a set of 6k icons that you can use in your projects and web pages.

I ho­ped that you will find them very us­able!

THank you a lot for your re­views!

**Comment by amyegan** [comment]
Good work on this icon gallery @IBNAHMADMOHAMED1! You should also share this with the Next.js community [on GitHub](https://github.com/vercel/next.js/discussions) or [on Discord](https://discord.com/invite/nextjs) to see what they think!

**Comment by emilkowalski** [comment]
Looking very nice @IBNAHMADMOHAMED1!
It would be cool if I'd be able to focus on the input by pressing `/`, this shortcut is commonly used in GitHub and Vercel for example.
You can also make your Select and Switcher components accessible by using something like https://www.radix-ui.com/ or https://headlessui.com/.
Additionally, adding a color picker would be a cool addition, so that I could select a color myself, instead of a pre-selected color sets/typing the hex value.
